engine-core/src/main/java/io/nosqlbench/engine/core/lifecycle/activity/ActivityExecutor.java

```diff
@@ -314,16 +316,16 @@ private void awaitAlignmentOfMotorStateToActivityState() {
             case Running:
                 tally.awaitNoneOther(RunState.Running, RunState.Finished);
                 break;
+            case Errored:
+            case Stopping:
             case Stopped:
-                tally.awaitNoneOther(RunState.Stopped, RunState.Finished);
+                tally.awaitNoneOther(RunState.Stopped, RunState.Finished, RunState.Errored);
```

Comment:
This may help for what I've been seeing on local runs!

Reply:
yes!